DEVELOPMENT.md

```diff
@@ -265,7 +266,7 @@ The recommended minimum development configuration is:
 
 <!-- TODO: Someone needs to validate the cluster-version-->
 1. Create a GKE cluster (with `--cluster-version=latest` but you can use any
-    version 1.18 or later):
```

Comment:
Any reason to revert that back ? on `main` (and most likely on 0.24 too), we have a minimum version of k8s at 1.18.

Reply:
Not sure how I reverted this... will put back.

Reply:
Done ;)